dll/win32/shell32/lang/fr-FR.rc

```diff
@@ -609,17 +609,17 @@ BEGIN
     AUTOCHECKBOX "Activer la compression", 28675, 16, 152, 155, 10
 END
 
-IDD_CHECK_DISK DIALOGEX 50, 50, 194, 115
+IDD_CHECK_DISK DIALOGEX 50, 50, 244, 115
 STYLE DS_SHELLFONT | DS_MODALFRAME | DS_CONTEXTHELP | WS_POPUPWINDOW | WS_VISIBLE | WS_CAPTION
 CAPTION "Vérifier le disque"
 FONT 8, "MS Shell Dlg"
 BEGIN
-    DEFPUSHBUTTON "Démarrer", IDOK, 63, 95, 60, 14
-    GROUPBOX "Options de vérification", -1, 7, 6, 179, 40
-    PUSHBUTTON "Annuler", IDCANCEL, 128, 95, 60, 14
-    AUTOCHECKBOX "Réparer automatiquement les erreurs du système de fichiers", 14000, 12, 15, 155, 10
-    AUTOCHECKBOX "&Scanner pour rechercher et réparer les secteurs défectueux", 14001, 16, 30, 165, 10
-    CONTROL "", 14002, "MSCTLS_PROGRESS32", 16, 7, 50, 179, 8
+    DEFPUSHBUTTON "Démarrer", IDOK, 113, 95, 60, 14
+    GROUPBOX "Options de vérification du disque", -1, 7, 6, 229, 40
+    PUSHBUTTON "Annuler", IDCANCEL, 178, 95, 60, 14
+    AUTOCHECKBOX "Réparer automatiquement les erreurs du système de fichiers", 14000, 12, 20, 215, 10
+    AUTOCHECKBOX "Rechercher et tenter une récupération des secteurs défectueux", 14001, 12, 30, 215, 10
```

Comment:
Please restore 4px offset (see comment from @HBelusca )

Reply:
either that, or remove that buggy offset in all translations.